webanno-constraints/src/main/java/de/tudarmstadt/ukp/clarin/webanno/constraints/model/FSFPair.java

```diff
@@ -86,12 +86,11 @@ else if (!featureStructure.equals(other.featureStructure)) {
     @Override
     public String toString()
     {
-        StringBuilder builder = new StringBuilder();
-        builder.append("FSFPair [featureStructure=");
-        builder.append(featureStructure);
-        builder.append(", affectedFeature=");
-        builder.append(affectedFeature);
-        builder.append("]");
-        return builder.toString();
+        String builder = "FSFPair [featureStructure=" +
```

Comment:
I am not sure the changes here improve the situation. It would probably be best to change this into a String.format(...) call... or at least to format the code to remove the many line breaks.